receiver/otlpreceiver/internal/arrow/arrow.go

```diff
@@ -49,14 +49,15 @@ type Receiver struct {
 	arrowpb.UnimplementedArrowStreamServiceServer
 
 	obsrecv       *obsreport.Receiver
-	arrowConsumer *batchEvent.Consumer
+	arrowConsumer arrowRecord.ConsumerAPI
```

Comment:
this is a nit and not entirely meant for this PR exactly, but isn't having API in the interface name redundant?

Reply:
https://github.com/f5/otel-arrow-adapter/issues/56